tests/integration/routes/account/patch-account-test.js

```diff
@@ -146,10 +146,65 @@ getServer(function (error, server) {
         t.end()
       })
     })
+    
+    group.test('username change', function (t) {
+      var couchdbChangeUserMock = nock('http://localhost:5984')
+        .post('/_users/_bulk_docs', function (body) {
+          return Joi.object({
+            all_or_nothing: Joi.boolean().only(true).required(),
+            docs: Joi.array().items(
+              Joi.object({
+                'name': Joi.string().only('org.couchdb.user:newusername').required(),
+                'renamedTo': Joi.string().only('newusername').required(),
+                'username': Joi.any().forbidden()
+              }),
+              Joi.object({
+                '_id': Joi.string().only('org.couchdb.user:oldusername').required(),
+                '_deleted': Joi.boolean().only(true).required()
```

Comment:
'renamedTo' shoudl be in the deleted `org.couchdb.user:oldusername` doc. Good idae with `'username': Joi.any().forbidden()`, can you add it to the 2nd document, too?